### DISCUSSION TITLE
addNewLine: false

### DISCUSSION THREAD

**Discussion by Akash9630** [xdan/jodit#960] [Q&A] [answered]
how to hide line break at the bottom of element when show line between cursor position?

**Comment by xdan** [comment, marked as answer]
That's the add-new-line helper — the line that lets you insert a paragraph between or after blocks. Turn it off with `addNewLine: false` in the config. If you only want to limit when it appears rather than kill it, `addNewLineTagsTriggers` controls which block tags trigger it.
